Subject: fix FTBFS on Debian GNU/Hurd
Origin: http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=532368#5
Bug-Debian: http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=532368
Forwarded: http://code.google.com/p/fqterm/issues/detail?id=268
Last-Update: 2009-11-11

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -71,6 +71,10 @@
   ENDIF(APPLE)
 ENDIF(WIN32)
 
+IF(CMAKE_SYSTEM_NAME MATCHES "GNU")
+  SET(SUB_CMAKELIST "CMakeLists.txt.linux")
+ENDIF(CMAKE_SYSTEM_NAME MATCHES "GNU")
+
 IF(SUB_CMAKELIST MATCHES "UNKNOWN")
   MESSAGE(FATAL_ERROR "Unsupported system type detected!")
 ELSE(SUB_CMAKELIST MATCHES "UNKNOWN")
